app/views/Export.js

```diff
@@ -129,27 +130,39 @@ export const ExportScreen = ({ navigation }) => {
 
           <ScrollView contentContainerStyle={styles.contentContainer}>
             <View style={styles.main}>
-              <Typography style={styles.exportSectionTitles}>
-                {t('label.tested_positive_title')}
-              </Typography>
-              <Typography style={styles.exportSectionPara}>
-                {t('label.export_para_1')}
-              </Typography>
-              <Typography style={styles.exportSectionPara}>
-                {t('label.export_para_2')}
+              <Typography use='headline2' style={styles.exportSectionTitles}>
+                {t('share.title')}
               </Typography>
+              <Trans i18nKey='share.paragraphs'>
+                <Typography
+                  use='body1'
+                  style={styles.exportSectionPara}
+                  // eslint-disable-next-line react-native/no-raw-text
+                >
+                  If you test positive for COVID-19, please do your part by
+                  sharing your location history with local authorities.
+                </Typography>
+                <Typography
+                  use='body1'
+                  style={styles.exportSectionPara}
+                  // eslint-disable-next-line react-native/no-raw-text
+                >
+                  Location is shared as a simple list of times and places, no
+                  additional information.
+                </Typography>
+              </Trans>
 
               <TouchableOpacity style={styles.exportButton} onPress={onShare}>
                 <Typography style={styles.exportButtonText}>
-                  {t('label.share_location_data')}
+                  {t('share.button_text')}
```

Comment:
Should this be 

```jsx
<Trans i18nKey='share.title>Share location data</Trans>
```

Reply:
I back tracked on `<Trans>` because we can end up in a situation where the react code can says `<Trans>Hey cool here's some text</Trans>`, but through some legal or other reason we have to change the english text, and this will read from the `en.json` file, and now there's a weird disconnect in the code and UI, especially for running unit tests.

What do you think?

Reply:
Yeah, that's a good point. Personally, when poking around the codebase it's a bit of a pain to have to keep `en.json` open and reference back to that all the time too though. 

Either way you are probably right about pulling back on the use of `<Trans>` though - I can live with a bit of a nuisance in development over a bug in prod